docs/source/releases/v1_14_0a0.rst

```diff
@@ -0,0 +1,66 @@
+.. dropdown:: Distribution Statement
+
+ | # # # This source code is protected under the license referenced at
+ | # # # https://github.com/NRLMMD-GEOIPS.
+
+Version 1.14.0a0 (2024-07-12)
+*****************************
+
+* Bug Fixes
+
+  * Fix failing test data CLI unit tests
+  * Correct typo causing product family xrdict_area_product_to_outlist failure
+* Documentation Updates
+
+  * Update LICENSE and DISTRIBUTION to latest approved version
+
+Bug Fixes
+=========
+
+Correct typo causing product family xrdict_area_product_to_outlist failure
+--------------------------------------------------------------------------
+
+*From GEOIPS#666: 2024-07-14, Final Bug Fixes Prior to Workshop*
+
+Product families that did not have pre-defined output lists and expected data
+passed unmodified from the readers to the output formatters were failing due to
+a missing else statement ensuring the list of final_products was set even when
+the output file list was not pre-defined.  Add else statement to set final_products
+to curr_products when output file list not defined.
+
+This resolved an error with product family xrdict_area_product_to_outlist when
+it was requested without sectoring, but it worked if sectoring was requested
+(followed a different route through the code for each).
```

Comment:
This is the fix for Coleman's issue.

Reply:
I meant for this to be on its own branch :-(

Reply:
moved this to its own branch.  Likely will autoclose the other PR if this one gets merged first.  Will clear out these changes from here if the other one gets merged first.